SimpleWAWS/Code/CsmExtensions/CsmResourceGroupExtensions.cs

```diff
@@ -403,9 +403,7 @@ private static bool IsSimpleWaws(CsmWrapper<CsmResourceGroup> csmResourceGroup)
 
             if (functionContainer == null || functionsStorageAccount == null) return; // This should throw some kind of error? maybe?
```

Comment:
You shouldn't have to worry about throwing here. You can throw at any point in the recourse group prep methods and it will always only affect that resource group only